src/formatDate.js

```diff
@@ -50,7 +50,56 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldDateArr = date.split(fromFormat[3]);
+  const newDateArr = [];
+  const objDate = {};
+
+  fromFormat.forEach((el, ind) => {
+    switch (true) {
+      case el.includes('YY'):
+        objDate.year = oldDateArr[ind];
+        break;
+      case el.includes('MM'):
+        objDate.month = oldDateArr[ind];
+        break;
+      case el.includes('DD'):
+        objDate.day = oldDateArr[ind];
+        break;
+      default:
+        objDate.delimiter = oldDateArr[ind];
+    }
+  });
+
+  let { year, delimiter } = objDate;
+  const { month, day } = objDate;
```

Comment:
```suggestion
  let { year } = objDate;
  const { month, day, delimiter } = objDate;
```
you don't change its value

Reply:
![image](https://github.com/mate-academy/js_format_date_advanced/assets/124107724/d01ce7fa-de73-4d3a-aa97-6dcc2d1fe11c)
But this value change here